cdk/lambdas/wdiv-s3-trigger/requirements/testing.txt

```diff
@@ -422,9 +422,9 @@ requests==2.31.0 \
     #   -c cdk/lambdas/wdiv-s3-trigger/requirements/../../../../requirements/constraints.txt
     #   moto
     #   responses
-responses==0.25.0 \
-    --hash=sha256:01ae6a02b4f34e39bffceb0fc6786b67a25eae919c6368d05eabc8d9576c2a66 \
-    --hash=sha256:2f0b9c2b6437db4b528619a77e5d565e4ec2a9532162ac1a131a83529db7be1a
+responses==0.25.3 \
+    --hash=sha256:521efcbc82081ab8daa588e08f7e8a64ce79b91c39f6e62199b19159bea7dbcb \
+    --hash=sha256:617b9247abd9ae28313d57a75880422d55ec63c29d33d629697590a034358dba
```

Comment:
I'm confused why `responses` got bumped here. Given the next thing we are going to do is convert this project to UV, I reckon lets not spend any time on this and just roll with it.

Reply:
Me too. Not sure why it got bumped, but had the same thought to just go with it, rather than unpick what had happened.